Jasper Overall/JSHOMWRK2/index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+  <head>
+    <title> HOMEWORK 1 </title>
+    <meta charset="UTF-8">
+    <<script type="t/Projects/seic38-homework/Jasper Overall/JSHOMWRK2/homework.jst">
```

Comment:
You've got an extra `<` here, also should be` src="filename"`. And you need to have `</script>` to close your script tag.